### ISSUE TITLE
Translations in folders are not imported

### ISSUE THREAD

**Issue by AlexKomMsk** [barryvdh/laravel-translation-manager#406]
I have a translation in a folder writer/messages.php 
This translation is not imported into the database. All translations in folders are not imported into the database. Translations without folders are successfully imported.

**Comment by HarryThe3rd** [comment]
The Problem is that the group pattern does not match with folders included in the translation string.
I have a fix for this and will create a pull request.

**Comment by simonjolsen** [comment]
So for me, the import from folders are now working, but publishing the translations in folders are not. It says it's successful, but it's not updating the file. If the file is not in a folder. It works.

**Comment by underdpt** [comment]
Yeah, I also have the issue that publishing translations in folders are not working. It works when downgrading to 0.6.2

**Comment by maxi032** [comment]
Fr me exporting the translations did not worked with wither 0.6.2 nor with 0.6. 3.
` php artisan translations:export --all
Done writing language files for ALL groups
`

**Comment by maxi032** [comment]
> For me exporting the translations did not worked with wither 0.6.2 nor with 0.6. 3. ` php artisan translations:export --all Done writing language files for ALL groups ` There was no change I am using Laravel 8.83

I had it working with v0.5.10 after I deleted compose.lock

**Comment by martinschenk** [comment]
i have the same issue. Not exporting to folders.
